qdrant-landing/content/documentation/embeddings/premai.md

```diff
@@ -0,0 +1,477 @@
+---
+title: Prem AI
+weight: 1000
+aliases: [ ../integrations/premai/ ]
+---
+
+# Prem AI
+
+[PremAI](https://premai.io/) is an all-in-one platform that simplifies the creation of robust, production-ready applications powered by Generative AI. By streamlining the development process, PremAI allows you to concentrate on enhancing user experience and driving overall growth for your application. You can quickly start using our platform [here](https://docs.premai.io/quick-start).
+
+Qdrant is compatible with Prem AI SDK (both Python and Javascript interfaces). We start by installing the SDKs. Here is how you install premai Python SDK. 
+
+```bash
+pip install premai
+```
+
+To install the npm package, type this command:
+
+```bash
+npm install @premai/prem-sdk 
+```
+
+### Import all required packages
+
+We start by importing all our required packages. Here is how you do that for python
+
+```python
+import os 
+import getpass
+from premai import Prem
+
+from qdrant_client import QdrantClient
+from qdrant_client.models import Distance, VectorParams
+
+if os.environ.get("PREMAI_API_KEY") is None:
+    os.environ["PREMAI_API_KEY"] = getpass.getpass("PremAI API Key:")
+```
+
+And here is how you do for javascript
+
+```jsx
+import Prem from '@premai/prem-sdk';
+import { QdrantClient } from '@qdrant/js-client-rest';
+```
+
+### Define all the constants
+
+After this we define our constants. For Prem, we are required to set our `project_id` and `embedding_model`. You can learn more about how to get your `project_id` and API Key from [here](https://docs.premai.io/quick-start). 
+
+From Qdrant’s side, we need to also define our server URL, in which we will be sending requests to do all sorts of CRUD operations to our vector database. We also need to define our collection name. You can learn more about these concepts from Qdrant’s [quick start](https://qdrant.tech/documentation/quick-start/) guide and [concepts](https://qdrant.tech/documentation/concepts/).
+
+Last but not the least, we also need to get our documents. For simplicity purpose, we define a small list of documents here. But in actual scenarios, this list should be derived from some source (like database or from an API call etc).
```

Comment:
Maybe make this a little less verbose.

Reply:
I see, can you please elaborate a bit?

Reply:
I updated this part, let me know if that works, thanks :)